qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -1007,6 +1007,10 @@ def run(
                 instance=qrt_options.instance, backend_name=qrt_options.backend
             )
             hgp_name = hgp.name
+        backend = self.backend(name=qrt_options.backend, instance=hgp_name)
+        status = backend.status()
+        if status.operational is True and status.status_msg != "active":
+            warnings.warn(f"The backend {backend.name} is currently paused.")
```

Comment:
Just noticed that the status message can also be `dedicated` or `internal` so we should probably only give this warning if the message is `maintenance`
<img width="404" alt="Screenshot 2023-06-06 at 10 35 46 AM" src="https://github.com/Qiskit/qiskit-ibm-runtime/assets/20911417/ba21dcde-199e-434e-bed5-a3607a724f8b">

Reply:
I think it would be good to give the warning for all these cases. Only instead of outputing 'no active' I should output the status message.